src/components/PastCommits.tsx

```diff
@@ -40,6 +42,7 @@ export class PastCommits extends React.Component<IPastCommitsProps, {}> {
           refresh={this.props.refresh}
           sideBarExpanded={this.props.sideBarExpanded}
           display={this.props.showList}
+          renderMime={this.props.renderMime}
```

Comment:
Yeah, now that I see how many places this is passed/used, definitely use context: https://reactjs.org/docs/context.html#when-to-use-context

Reply:
Yeah I would second this. It would be great to remove some of these props we pass down the whole hierarchy and put them on context.

Reply:
TIL ReactContext :-) 

My latest commit addresses this.